app/Models/Revision.php

```diff
@@ -70,6 +70,7 @@ protected static function boot()
 	public function setPublished()
 	{
 		$this->published_at = $this->published_at ? $this->published_at : Carbon::now();
```

Comment:
@samyapp is there any reason why this doesn't always update the 'published_at' to now()?